RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate methods/getFileUrlFromMessage to typescript

## Proposed changes

Migrate methods/getFileUrlFromMessage to typescript

## Issue(s)	


## How to test or reproduce
On a chat, open the actions, select files, then verify if the files are loaded correctly.


## Screenshots

## Types of changes




## Checklist



## Further comments
